### ISSUE TITLE
UINavigationBar back button has no left padding on iPhone 14

### ISSUE THREAD

**Issue by mournblade** [futuretap/InAppSettingsKit#484]
Please see attached image. For some reason the back button hugs the left wall on my iPhone 14 pro. 

This started happening for us recently and it doesn't happen on for example iPhone X. Any idea why this happens? As you can see, the table rows themselves have padding.

<img width="470" alt="Screenshot 2023-08-23 at 12 16 03" src="https://github.com/futuretap/InAppSettingsKit/assets/3775802/aaac30cb-8656-4ab9-af6e-44b5f506db95">

**Comment by futuretap** [comment]
Can you reproduce this with the IASK sample app? We can’t, at least in the simulator.

**Comment by mournblade** [comment]
No, the sample app works. I have the tableview in a tab, but I took that code from your tab example, so I have no idea what it might be. It's so strange that it only happens on the newer phones.
